pkg_defs/astropy-healpix/meta.yaml

```diff
@@ -0,0 +1,54 @@
+# Recipe largely borrowed from https://github.com/conda-forge/astropy-healpix-feedstock
+# but the build and run requirements have been set manually to ska3 requirements
+
+{% set name = "astropy-healpix" %}
+{% set version = "0.4" %}
+{% set sha256 = "8c9709ac923759c92eca6d2e623e734d0f417eed40ba835b77d99dec09e51aa2" %}
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 1001
+  script: python setup.py install --single-version-externally-managed --record record.txt --offline --no-git
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy ==1.12.1
+    - llvmlite ==0.18.0
+    - gcc ==4.8.5 # [linux64]
+    - libgcc ==4.8.5 # [linux64]
+    - astropy
+  run:
+    - python
+    - llvmlite ==0.18.0
+    - numpy ==1.12.1
+    - astropy >=2.0
+    - libgcc ==4.8.5
+    - astropy >=2.0
```

Comment:
Repeat.

Reply:
Oops, I cut the wrong one to have github pick this up as outdated.